src/OpenTelemetry/Metrics/HistogramBuckets.cs

```diff
@@ -24,6 +26,8 @@ namespace OpenTelemetry.Metrics
     // Note: Does not implement IEnumerable<> to prevent accidental boxing.
     public class HistogramBuckets
     {
+        internal const int DefaultHistogramCountForBinarySearch = 50;
```

Comment:
Rename to `DefaultBucketCountForBinarySearch`.

Reply:
Renamed. Technically it is the boundary count so I named as such. If we have 50 buckets then there are 49 boundaries.

i.e. if we have one boundary [ 1 ] we get buckets from -inf to 0 and 0 to +inf.